common/config/rush/command-line.json

```diff
@@ -37,7 +37,7 @@
       "summary": "Switch flavor to beta",
       "description": "Every build and test command will run based on beta version.",
       "autoinstallerName": "rush-cpy",
-      "shellCommand": "rush install && cpy common/config/env/beta.env common/config/env --rename .env && rush rebuild",
+      "shellCommand": "rush install && cpy common/config/env/beta.env common/config/env --rename .env",
```

Comment:
I see why you removed `&& rush rebuild` here -- so that CI won't do an extra rebuild when switching flavors.

But that step was necessary to ensure that developers don't accidentally use previously built packlets after switch flavors, right?

Perhaps add a rush command that deletes all the `.rush/temp` folders and run that so that next build will necessary build everything from scratch? See https://rushjs.io/pages/commands/rush_build/

Reply:
Done! This is actually a good idea! So we can still use rush build, but build everything in fresh